gemsiv/delaunay/quadedge.C

```diff
@@ -1,4 +1,9 @@
 #include "quadedge.h"
+#include "../../dummy.h"
```

Comment:
This include may be needed, but seems a bit odd. Could you explain "dummy.h" and what it's for? It looks a bit catch-all.